guides/build-api-plugin.md

```diff
@@ -225,6 +225,28 @@ query {
 
 We’ve done everything we need to add volume data to our backpacks and query that information across our store. That’s useful to Leader of the Pack’s store administrators, but we also need to get that information to customers. This will require some design work on our product pages, which are dependent on the storefront plugin that we’re using. Generally, this will involve making sure that our product query includes the new `volume` field and that the information is passed to the `render()` function in the file that describes our product detail page. For more information, see the [Creating and Organizing Products doc](/developer/open-commerce/docs/creating-organizing-products/).
 
+## Link multiple packages with package configuration file
```

Comment:
I feel like this doesn't quite make sense within the context of what this doc is talking about. It's about a dev process (is the `package-link` script referenced in this doc). This seems like it would make more sense in the Getting Started doc

Reply:
It has a "Link the plugin" section which talks about package-link for a single package.